app/application/usecases/auth/signup_usecase.py

```
@@ -0,0 +1,10 @@
+from app.application.services.auth import SignUpService
+from app.interface.schemas.auth.requests import SignUpRequest
+
+
+class SignUpUseCase:
+    def __init__(self, signup_service: SignUpService):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)